src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -2067,6 +2101,61 @@ private void LowerBoundTypeArgumentInference(NamedTypeSymbol source, NamedTypeSy
             targetTypeArguments.Free();
         }
 
+#nullable enable
+        private bool LowerBoundFunctionPointerTypeInference(TypeSymbol source, TypeSymbol target, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            if (source is not FunctionPointerTypeSymbol { Signature: { } sourceSignature } || target is not FunctionPointerTypeSymbol { Signature: { } targetSignature })
```

Comment:
>target is not FunctionPointerTypeSymbol { Signature: { } targetSignature } [](start = 94, length = 74)

I am not sure where we deal with the conversion part from the specification: "such that `U` has an implicit function pointer conversion to `delegate*<U2..Uk, U1>`"

Reply:
The specification no longer has that section.

---
In reply to: [558412661](https://github.com/dotnet/roslyn/pull/50249#discussion_r558412661) [](ancestors = 558412661)